NAS2D/Resource/Font.cpp

```diff
@@ -66,7 +66,7 @@ namespace
  * Instantiate a Font using a TrueType or OpenType font.
  *
  * \param	filePath	Path to a font file.
- * \param	ptSize		Point size of the font. Defaults to 12pt.
+ * \param	ptSize		Point size of the font.
```

Comment:
For example, this looks to be removing an obsolete and now incorrect comment. This should ideally stand out from the other changes and be explained with a commit message.